assets/dev/js/admin/admin-feedback.js

```diff
@@ -52,15 +52,13 @@
 								callback: self.sendFeedback.bind( self ),
 							} );
 
-							if ( ! elementorAdmin.config.feedback.is_tracker_opted_in ) {
```

Comment:
Maybe we also should delete the `is_tracker_opted_in` from the config if we no longer using it?